### DISCUSSION TITLE
socket.io-client   Build issue in Angular7.1 and Typescript 3.1.6

### DISCUSSION THREAD

**Discussion by rsurendraidexcel** [socketio/socket.io#4251] [Q&A]
Hi I have  got issue  while run the angular project
![Error](https://user-images.githubusercontent.com/7713535/149530374-851b68a6-e3bd-47be-9a4f-4b521f9d9ad0.jpg)
  
if any Idia  about this kind of issue .
 
Angular7.1 verion and Typescript 3.1.6

**Comment by darrachequesne** [comment]
`typescript@3.1.6` is quite an old version: https://www.npmjs.com/package/typescript/v/3.1.6 (published 3 years ago)

Could you please try with a more recent version?
